fvm/src/engine/mod.rs

```diff
@@ -164,6 +166,7 @@ fn wasmtime_config(ec: &EngineConfig) -> anyhow::Result<wasmtime::Config> {
 
     // wasmtime default: true
     // We don't want threads, there is no way to ensure determinism
+    #[cfg(feature = "wasmtime/threads")]
     c.wasm_threads(false);
```

Comment:
should we just remove the feature then? or are you wanting to keep this just to be explicit about it for if it eventually shows up not behind a feature flag?

Reply:
ah, it's on by default anyway so always tehre